src/ResourceManager/DevSpaces/Commands.DevSpaces/help/AzureRM.DevSpaces.md

```diff
@@ -0,0 +1,107 @@
+## Service Release Details
```

Comment:
This isn't right - please generate this file using Update-MarkdownHelpModule: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Reply:
Done